test/csuite/wt4156_metadata_salvage/main.c

```diff
@@ -518,13 +526,12 @@ main(int argc, char *argv[])
      * We need to set up the string before we clean up the structure. Then after the clean up we
      * will run this command.
      */
-    testutil_check(__wt_snprintf(buf, sizeof(buf), "rm -rf core* %s*", home));
+    testutil_check(__wt_snprintf(buf, sizeof(buf), "rm -rf %s.%s", home, SAVE));
```

Comment:
I think this changes the cleanup. The original code cleaned up `<home>*` which would get both $DIR and $DIR.SAVE. Now you're only removing $DIR.SAVE and leaving $DIR. I think this should just remove the `core` piece.

Reply:
It turns out that `testutil_cleanup()` already deletes `home`.  (Well, it deletes `opts->home`. But `home` is just a cached global copy of that.) So all that's left to clean up is $DIR.SAVE.

Reply:
Ok. Want to add a comment there to that effect so I'll remember later! I've dabbled in this test quite a bit and clearly didn't remember that. Thanks!

Reply:
Done!